src/stores/PersonenkontextStore.ts

```diff
@@ -242,7 +242,7 @@ export const usePersonenkontextStore: StoreDefinition<
       this.loading = true;
       try {
         const { data }: { data: DBiamPersonenkontextResponse } =
-          await dbiamPersonenkontexteApi.dBiamPersonenkontextControllerCreatePersonenkontext(personenkontext);
+          await dbiamPersonenkontexteApi.dBiamPersonenkontextControllerCreatePersonenkontextMigration(personenkontext);
```

Comment:
Das habe Ich in SPSH-1089, weil er nur für die Migration benutzt wird. Aber ändert hier nix.

Reply:
Es gab einen build Fehler, wenn es nicht gemacht wird.

Reply:
Muss hier dann eine andere Methode verwendet werden, wenn diese nur für die Migration ist? Das war nach dem Update mit main auf einmal im Branch.

Reply:
Nein, das wird rausfliegen. Wir nutzen den Endpunkt gar nicht mehr. Der war für die Anlage der einzelnen Kontexten bedacht aber das brauchen wir nicht mehr.